seg7.py

```diff
@@ -0,0 +1,108 @@
+import RPi.GPIO as GPIO
+import time
+
+GPIO.setmode(GPIO.BCM)
+GPIO.setwarnings(False)
+
+#GPIO ports for the 7seg pins
+segment4 = (26,20,21,16)
+
+for segment in segment4:
+    GPIO.setup(segment, GPIO.OUT)
+    GPIO.output(segment, 0)
+
+#Digit 1
+GPIO.setup(5, GPIO.OUT)
+GPIO.output(5, 0) #Off initially
+#Digit 2
+GPIO.setup(6, GPIO.OUT)
+GPIO.output(6, 0) #Off initially
+
+null = [0,0,0,0]
+zero = [0,0,0,0]
+one = [0,0,0,1]
+two = [0,0,1,0]
+three = [0,0,1,1]
+four = [0,1,0,0]
+five = [0,1,0,1]
+six = [0,1,1,0]
+seven = [0,1,1,1]
+eight = [1,0,0,0]
+nine = [1,0,0,1]
+
+
+def print_segment(num):

```

Comment:
This can be refactored to look cleaner. You can put lists zero - nine into another list (list of lists essentially) and use num as an index. This way it will avoid all the if statements to check the value of num and calling the correct list.